CMP1903M Assessment 1 Base Code/Input.cs

```diff
@@ -32,16 +32,37 @@ public List<string> manualTextInput()
                     this.textList.Add(text); 
                 }
             } while (!text.EndsWith("*")) ;
-                return textList;
+                
+            string lastSentence = textList[textList.Count - 1];
+            textList[textList.Count-1] = lastSentence.Remove(lastSentence.Length - 1);
+            return textList;
         }
 
         //Method: fileTextInput
         //Arguments: string (the file path)
-        //Returns: string
+        //Returns: file contents in a list
         //Gets text input from a .txt file
-        public List<string> fileTextInput(string fileName)
+        public List<string> fileTextInput()
         {
-
+            
+            try
```

Comment:
Error handling is implemented which is good, might be worth commenting to point it out as per the brief